code/game/objects/structures/ladders.dm

```diff
@@ -209,3 +209,47 @@
 
 /obj/structure/ladder/crafted
 	crafted = TRUE
+
+/obj/structure/ladder/unbreakable/binary
+	name = "mysterious ladder"
+	desc = "Where does it go?"
+	height = 0
+	id = "lavaland_binary"
+	var/area_to_place = /area/lavaland/surface/outdoors
+	var/active = FALSE
+
+/obj/structure/ladder/unbreakable/binary/proc/ActivateAlmonds()
```

Comment:
so the tear will potentially create a portal to elsewhere on the map?

Reply:
The tear creates a large chasm. This can be plated over, so no worry about that, however when you fall you are teleported to a tear in the fabric of reality on the centcom Z-level. That has a ladder that leads you somewhere to lavaland in it.

Reply:
sounds interesting

Reply:
yeah pretty much what he said